packages/nouns-sdk/src/image/svg-builder.ts

```diff
@@ -54,6 +54,7 @@ export const buildSVG = (
     let currentY = bounds.top;
 
     rects.forEach(draw => {
+      // eslint-disable-next-line prefer-const
```

Comment:
Nit: Fix rather than silencing error. e.g.

```typescript
let drawLength = draw[0];
const colorIndex = draw[1];
```

Reply:
resolved in https://github.com/nounsDAO/nouns-monorepo/pull/595/commits/dc4f60b4c037d1eb25377ce8e4986324ee243062